doc/cygnus-ngsi-ld/installation_and_administration_guide/introduction.md

```diff
@@ -9,13 +9,9 @@ Internally, cygnus-ngsi-ld is based on [Apache Flume](http://flume.apache.org/),
 
 Current stable release is able to persist Orion context data in:
 
-* [HDFS](http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html), the [Hadoop](http://hadoop.apache.org/) distributed file system.
-* [MySQL](https://www.mysql.com/), the well-know relational database manager.
-* [CKAN](http://ckan.org/), an Open Data platform.
-* [MongoDB](https://www.mongodb.org/), the NoSQL document-oriented database.
-* [STH Comet](https://github.com/telefonicaid/IoT-STH), a Short-Term Historic database built on top of MongoDB.
-* [Kafka](http://kafka.apache.org/), the publish-subscribe messaging broker.
-* [DynamoDB](https://aws.amazon.com/dynamodb/), a cloud-based NoSQL database by [Amazon Web Services](https://aws.amazon.com/).
+* [PostgreSQL](http://www.postgresql.org/), the well-know relational database manager.
+* [PostGIS](http://postgis.net/), a spatial database extender for PostgreSQL object-relational database.
+* [CKAN](http://ckan.org/), an Open Data platform. 
```

Comment:
I understand you have reduced the list to only the sinks that are supported by cygnus-ngsi-ld. Is my understanding correct?

Reply:
Yes, this is specific to the new cygnus-ngsi-ld agent

Reply:
Ok. NTC.